include/sys/time.h

```diff
@@ -160,6 +160,17 @@ struct itimerval {
 
 #ifdef _KERNEL
 
+/* It allows to extend the capacity
+   for counters with 16/32 bits, . */
+typedef union {
+  /* assumes little endian order */
+  struct {
+    uint32_t lo;
+    uint32_t hi;
+  };
+  uint64_t val;
+} counter_t;
```

Comment:
This type is globally visible in the kernel after inclusion of `sys/time.h`. For that reason I think that the typename is too ambiguous. That would be ok if defined in `_.c` file, but not in a header file.

Reply:
Done.